### PR TITLE
[NCL-6030] Remove testsuite activation by profile

### PR DESCRIPTION
**Pull request by jomrazek**
### Checklist:

* [ ] Have you added a note in the CHANGELOG.md for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by jomrazek** [comment]
:exclamation::exclamation::exclamation:When merged change jenkins build script to
`mvn clean install -DtestsuiteContainer=$PATH_TO_EAP -Dtestsuite.test.excludes=**/BugReporoducerRemoteTestIT.java`

File: pom.xml [hunk 1]
```diff
@@ -424,22 +425,45 @@
...
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
 
     </build>
     <profiles>
         <profile>
-            <id>testsuite</id>
+            <id>excludeTests</id>
```
Comment (matejonnet) [inline pom.xml:445]: What's the use case of this profile ?

Comment (jomrazek) [reply pom.xml:445]: To exclude any test, it's used in jenkins for example (not sure if still needed) I left it as it was just moved it so remote test exclusion is actually working

Comment (matejonnet) [reply pom.xml:445]: Don't know about this use case but in general if the test is failing it's better to ignore it on the test source level.
If it's special type of test that needs to be excluded by default an option is to create a "group" and exclude it.

It's good to have a project in s state that it builds and all the tests pass for anyone with standard `mvn clean install` command.

Comment (jomrazek) [reply pom.xml:445]: I have no problem to ignore class that's excluded in jenkins and remove this profile, but as I said I don't know reasoning behind this approach

**Comment by jomrazek** [comment]
:exclamation:  remove 
`-Dtestsuite.test.excludes=**/BugReporoducerRemoteTest.java` from jenkins PR job after merge

@janinko you might comment on tests that were excluded in past if you have some concerns about ignoring them now

**Comment by janinko** [comment]
>  remove
> `-Dtestsuite.test.excludes=**/BugReporoducerRemoteTest.java` from jenkins PR job after merge
> 
> @janinko you might comment on tests that were excluded in past if you have some concerns about ignoring them now

The test was ingored only in PRs as it took long time to run. I'm not sure if that's still the case when we use wiremock, maybe we should try to un-ignore the test.

**Comment by jomrazek** [comment]
@janinko those excluded tests are failing now

**Comment by jomrazek** [comment]
Ignored tests, they were not performed from beginning and I think is out of scope of this issue to fix them when they're failing

**Review by janinko — APPROVED** [review]

**Pull request merged**